ui/pages/home/home.component.js

```diff
@@ -946,6 +954,28 @@ export default class Home extends PureComponent {
             }
           </div>
           {this.renderNotifications()}
+          {showSurveyToast ? (
+            <BannerBase
+              className="home__survey-banner"
+              backgroundColor={Color.iconDefault}
```

Comment:
You can update the entire component to use dark mode colors by using the data-theme attribute. I beleive it should be using dark theme colors no matter the theme
```suggestion
              data-theme="dark"
              backgroundColor={Color.iconDefault}
```

Reply:
Updated here [ae964c9](https://github.com/MetaMask/metamask-extension/pull/21918/commits/ae964c94f250ba4f82df38ebb5861dd362c8a935)